lib/membrane/video_compositor/video_compositor.ex

```diff
@@ -84,6 +84,8 @@ defmodule Membrane.LiveCompositor do
 
   alias Membrane.{Opus, Pad, RemoteStream, RTP, TCP}
 
+  alias Membrane.LiveCompositor
```

Comment:
Aliasing the same module seems weird. You can either alias `EventHandler` directly or use `__MODULE__`